python/ray/serve/_private/http_proxy.py

```diff
@@ -703,6 +703,14 @@ async def health_response(self, proxy_request: ProxyRequest) -> ProxyResponse:
             response=response_proto.SerializeToString(),
         )
 
+    def internal_error_response(
```

Comment:
should this function being a util func?  look like not related to the instance of `gRPCProxy`.

Reply:
I think this should stay within `gRPCProxy` it can only be used when responding gRPC request as well. But will also give it a `_` :)